zerver/tests/test_logging_handlers.py

```diff
@@ -22,22 +22,19 @@
 ] = None
 
 
-def capture_and_throw(domain: Optional[str] = None) -> Callable[[ViewFuncT], ViewFuncT]:
-    def wrapper(view_func: ViewFuncT) -> ViewFuncT:
-        @wraps(view_func)
-        def wrapped_view(request: HttpRequest, *args: object, **kwargs: object) -> NoReturn:
-            global captured_request
-            captured_request = request
-            try:
-                raise Exception("Request error")
-            except Exception as e:
-                global captured_exc_info
-                captured_exc_info = sys.exc_info()
-                raise e
-
-        return cast(ViewFuncT, wrapped_view)  # https://github.com/python/mypy/issues/1927
+def capture_and_throw(view_func: ViewFuncT) -> ViewFuncT:
+    @wraps(view_func)
+    def wrapped_view(request: HttpRequest, *args: object, **kwargs: object) -> NoReturn:
+        global captured_request
+        captured_request = request
+        try:
+            raise Exception("Request error")
+        except Exception as e:
+            global captured_exc_info
+            captured_exc_info = sys.exc_info()
+            raise e
 
-    return wrapper
+    return cast(ViewFuncT, wrapped_view)  # https://github.com/python/mypy/issues/1927
```

Comment:
Note: We are leaving this alone for now. We will replace `ViewFuncT` here and there in a separate PR in the future.